app/views/agency_update_mailer/new_agency_update_email.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <p>You got a new organization update request from <%= @agency.name %>!</p>
+    <p>
+    Update Request Details<br>
+    --------------------------
+    </p>
+    <p>Organization Name: <%= @agency.name %></p>
+    <p>Submitted by: <%= @aur.submitted_by %></p>
+    <p>Phone: <a href="tel:<%=@aur.phone%>"><%= @aur.phone %></a></p>
+    <p>Email: <a href = "mailto:<%=@aur.submitter_email%>"><%= @aur.submitter_email %></a></p>
```

Comment:
Add a link to the update request so that they can just click on it and not have to search for it.